src/main/java/uk/gov/hmcts/reform/wataskmanagementapi/domain/entities/camunda/MappedTask.java

```diff
@@ -0,0 +1,175 @@
+package uk.gov.hmcts.reform.wataskmanagementapi.domain.entities.camunda;
+
+import com.fasterxml.jackson.annotation.JsonFormat;
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.databind.PropertyNamingStrategy;
+import com.fasterxml.jackson.databind.annotation.JsonNaming;
+import io.swagger.annotations.ApiModel;
+import org.springframework.format.annotation.DateTimeFormat;
+
+import static uk.gov.hmcts.reform.wataskmanagementapi.domain.entities.camunda.CamundaTime.CAMUNDA_DATA_TIME_FORMAT;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+@JsonNaming(PropertyNamingStrategy.SnakeCaseStrategy.class)
+@SuppressWarnings({"PMD.LawOfDemeter","PMD.TooManyFields","PMD.ExcessiveParameterList"})
+@ApiModel("Task")
+public class MappedTask {
+
+    private String id;
+    private String name;
+    private String assignee;
+    private String type;
+    private String taskState;
+    private String taskSystem;
+    private String securityClassification;
+    private String taskTitle;
+    @DateTimeFormat(iso = DateTimeFormat.ISO.DATE)
+    @JsonFormat(pattern = CAMUNDA_DATA_TIME_FORMAT)
+    private String createdDate;
+    @DateTimeFormat(iso = DateTimeFormat.ISO.DATE)
+    @JsonFormat(pattern = CAMUNDA_DATA_TIME_FORMAT)
+    private String dueDate;
+    private String locationName;
+    private String location;
+    private String executionType;
+    private String jurisdiction;
+    private String region;
+    private String caseTypeId;
+    private String caseId;
+    private String caseCategory;
+    private String caseName;
+    private Boolean autoAssigned;
+
+    public MappedTask() {
```

Comment:
We have said we will document this interface for CCD well. You can do that by adding annotations to this class which get rendered on the swagger page. See CreateTaskRequest in wa-workflow-api and the @ApiModelProperty annotation. We need to describe each field and also say any restrictions on the value i.e. give the format for the strings returned by dueDate etc.